include/receivercadmium.hpp

```diff
@@ -0,0 +1,115 @@
+/**
+* Cristina Ruiz Martin
+* ARSLab - Carleton University
+*
+*/
+
+#ifndef __BOOST_SIMULATION_PDEVS_RECEIVER_HPP__
+#define __BOOST_SIMULATION_PDEVS_RECEIVER_HPP__
+
+
+#include <cadmium/modeling/ports.hpp>
+#include <cadmium/modeling/message_bag.hpp>
+#include <limits>
+#include <math.h> 
+#include <assert.h>
+#include <memory>
+#include <iomanip>
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <chrono>
+#include <algorithm>
+#include <limits>
+#include <random>
+
+#include "message.hpp"
+
+using namespace cadmium;
+using namespace std;
+
+//Port definition
+struct receiver_defs{
+    struct output : public out_port<Message_t> {
+    };
+    struct input : public in_port<Message_t> {
+    };
+};
+   
+template<typename TIME>
+class Receiver{
+    using defs=receiver_defs; // putting definitions in context
+    public:
+    //Parameters to be overwriten when instantiating the atomic model
+    TIME   PREPARATION_TIME;
+    // default constructor
+    Receiver() noexcept{
+    	PREPARATION_TIME  = TIME("00:00:10");
```

Comment:
change the proper spacing before and after **=** sign.

Reply:
Is it mandatory?